package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node ./api/index.js",
+    "nodemon": "./node_modules/nodemon/bin/nodemon.js ./api/index.js",
```

Comment:
A to tu trzeba bezposrednio odpalac?
Nie mozna zrobic: nodemon index.js
Zrób tak jak sie da.

Reply:
Tak działa bezproblemowo

Reply:
Super.